build.sbt

```diff
@@ -47,10 +48,10 @@ lazy val root = (project in file("."))
     PlayKeys.playDefaultPort := 10701,
     ScoverageKeys.coverageExcludedFiles := "<empty>;Reverse.*;.*handlers.*;.*components.*;" +
       ".*Routes.*;.*viewmodels.govuk.*;",
-    ScoverageKeys.coverageMinimumStmtTotal := 78,
+    ScoverageKeys.coverageMinimumStmtTotal := 83,
```

Comment:
Not necessarily now, but gradually we'd need to go back to 90 here

Reply:
I made it 78 back 👍

Reply:
That's fine, we can monitor if for a while in the pipeline and we can change it back later to 90 when we are confident we are continuously above 90.